assets/src/edit-story/components/canvas/singleSelectionMovable.js

```diff
@@ -169,6 +171,16 @@ function SingleSelectionMovable({ selectedElement, targetEl, pushEvent }) {
     !snapDisabled && (!isDragging || (isDragging && !activeDropTargetId));
   const hideHandles = (isDragging && isMaskable) || Boolean(draggingResource);
 
+  const wasElementDeleted = (target) => {
```

Comment:
This is a bit weirdly named. Maybe split into a function asking if it should be deleted and another actually deleting it.

Reply:
Hmm, `isTargetOutOfContainer` is basically the function asking if it should be deleted.

Perhaps there's a better name, e.g. `handleElementOutOfCanvas`? Changed it to this, let me know if this makes more sense.